content/docs/setup/kubernetes/download-release/index.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: Download the Istio release
+title: Downloading the Release
 description: Instructions to download the Istio release.
-weight: 90
+weight: 2
 keywords: [kubernetes]
 ---
 
```

Comment:
In a separate PR: maybe we can remove the 'getLatestIstio' and stick with downloading from release page.
Running shell scripts from the web is not best security practice.